### PR TITLE
Feature cleanup

### PR DESCRIPTION
**Pull request by vkarlen**
Cleaned up and organized all files

### CODE REVIEW

File: src/components/ShippingInfo/ShippingInfo.jsx [hunk 1]
```diff
@@ -55,21 +54,22 @@ function ShippingInfo() {
...
       });
+
+      // TO DO - Change to styled alert
       alert('Shipping Successful!');
+
       history.push('/samples');
     }
-  } // end handleSubmit
-
-  const backBtn = () => {
```
Comment (michaelshaleen) [inline src/components/ShippingInfo/ShippingInfo.jsx:57]: Why are we nixing this function?

Comment (vkarlen) [reply src/components/ShippingInfo/ShippingInfo.jsx:57]: The only thing it needs to do is a history.push, so I just moved that inline on the onClick

Comment (vkarlen) [reply src/components/ShippingInfo/ShippingInfo.jsx:57]: (line 127)

Comment (vkarlen) [reply src/components/ShippingInfo/ShippingInfo.jsx:57]: (line 127)

**Review by michaelshaleen — APPROVED** [review]

**Pull request merged**